packages/nuxt-locokit/src/runtime/tailwind.config.ts

```diff
@@ -31,6 +31,7 @@ export default {
         'warning-lighten': 'var(--color-warning-lighten)',
         'warning-light': 'var(--color-warning-light)',
         'warning-dark': 'var(--color-warning-dark)',
+        lck: 'var(--text-color)',
       },
       borderRadius: {
         lck: 'var(--border-radius)',
```

Comment:
Same as above, sounds strange to have `lck` here.

Prefer `text` for the color, maybe `default` for the border radius ?

Reply:
Same, in class it will be `rounded-lck`

Did I still need to rename it ,

Reply:
Mmm... I think we need to discuss a little bit how we define those colors.
Maybe in another issue.

When I read https://tailwindcss.com/docs/customizing-colors#using-custom-colors, we can use an object notation allowing light / dark shades.

For border radius, I think a `default` value could be a good path, but... need to be discussed.

Don't update the code, I open a new issue : https://github.com/locokit/locokit/issues/224